src/catalog/catalog_entry/table_catalog_entry.cpp

```diff
@@ -26,13 +26,19 @@
 #include "duckdb/common/field_writer.hpp"
 #include "duckdb/main/client_context.hpp"
 #include "duckdb/catalog/catalog_entry/type_catalog_entry.hpp"
+#include "duckdb/function/table/table_scan.hpp"
```

Comment:
Are these extra includes necessary here?

Reply:
I've removed the following includes here:

```
#include "duckdb/main/connection.hpp"
#include duckdb/planner/expression/bound_constant_expression.hpp"
#include "duckdb/common/field_writer.hpp"
#include "duckdb/main/client_context.hpp"
#include "duckdb/catalog/catalog_entry/type_catalog_entry.hpp"
#include "duckdb/function/table/table_scan.hpp"
#include "duckdb/planner/operator/logical_get.hpp"
```